tox.ini

```diff
@@ -18,9 +18,12 @@ deps = -rrequirements-test.txt
 [testenv:cover]
 passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
 basepython = python
+setenv =
+    THRIFTRW_COVERAGE=1
```

Comment:
Yeah. It appears that cython method signatures are being counted as uncovered (probably because they raise exceptions if they're called with incorrect types). For example, https://coveralls.io/builds/4006227/source?filename=thriftrw%2Fprotocol%2Fbinary.pyx